### PR TITLE
Add method `BultinRunner::get_used_diluted_check_units`

### PR DESCRIPTION
**Pull request by fmoletta**
Add method `BultinRunner::get_used_diluted_check_units()`
Add unit tests

### CODE REVIEW

File: src/vm/runners/builtin_runner/bitwise.rs [hunk 1]
```diff
@@ -98,6 +98,24 @@ impl BitwiseBuiltinRunner {
...
+                };
+            }
+        }
+        let partition_lengh = partition.len();
+        let num_trimmed = partition
+            .into_iter()
+            .filter(|elem| elem + diluted_spacing * (diluted_n_bits - 1) + 1 > total_n_bits)
+            .count();
+        4 * partition_lengh + num_trimmed
+    }
```
Comment (Oppen) [inline src/vm/runners/builtin_runner/bitwise.rs:118]: I think this merits a comment explaining what it does. It's certainly not obvious.

Comment (fmoletta) [reply src/vm/runners/builtin_runner/bitwise.rs:118]: The only documentation for this method is "Returns the number of diluted check units used by the builtin." in the class' method definition, so Im not really sure what it does either

Comment (Oppen) [reply src/vm/runners/builtin_runner/bitwise.rs:118]: There's a tiny bit more context in the callers of this function, apparently those "diluted check units" are extra zeros in between information carrying bits. I'm guessing some error correcting code gets injected in those at some later point.

**Review by Oppen — APPROVED** [review]
The code seems equivalent to that of Cairo. It's sad that there's not much information about the intent, but LGTM.

**Review by juanbono — APPROVED** [review]

**Pull request merged**